config/custom-environment-variables.json

```diff
@@ -23,7 +23,11 @@
   },
   "draft-store": {
     "url": "DRAFT_STORE_URL",
-    "apiVersion": "DRAFT_STORE_API_VERSION"
+    "apiVersion": "DRAFT_STORE_API_VERSION",
+    "secrets": {
+      "primary": "DRAFT_STORE_SECRET_PRIMARY",
```

Comment:
what does this have to do with document upload?

Reply:
we upgraded draft store version here